atom/common/options_switches.cc

```diff
@@ -113,6 +113,10 @@ const char kDisableHttpCache[] = "disable-http-cache";
 // Register schemes to standard.
 const char kRegisterStandardSchemes[] = "register-standard-schemes";
 
+// The minimum SSL/TLS version ("tls1", "tls1.1", or "tls1.2") that
+// TLS fallback will accept.
+const char kSSLVersionFallbackMin[] = "ssl-version-fallback-min";
```

Comment:
Can you verify whether `ssl-version-fallback-min` does the job? I don't mind which one is more correct, but the OP only verified `ssl-version-max` works and I'm not sure whether `ssl-version-fallback-min` does.